### PR TITLE
RadioGroup.onContentReady should rise after changing dataSource (close T697809)

### PR DESCRIPTION
**Pull request by churkin**

### CODE REVIEW

File: js/ui/radio_group/radio_group.js [hunk 1]
```diff
@@ -255,9 +257,12 @@ var RadioGroup = Editor.inherit({
         return this._$submitElement;
     },
 
-    _contentReadyHandler: function() {
+    _contentReadyHandler: function(e) {
+        this._radios = this._radios || e.component;
```
Comment (lukyanovas) [inline js/ui/radio_group/radio_group.js]: For what?

Comment (churkin) [reply js/ui/radio_group/radio_group.js]: Why I should store `this._radios`?

Comment (lukyanovas) [reply js/ui/radio_group/radio_group.js]: RadioGroup uses an instance of RadioCollection. And I see that you keep on storing `this._radios`. I believe that if you want to store something( e.g.  `this._radios`) the best place to save it is its creation.

**Comment by churkin** [comment]
Ready to review

File: js/ui/radio_group/radio_group.js (hunk 1 above)
Comment (churkin) [reply js/ui/radio_group/radio_group.js]: In your case at the first time `_contentReadyHandler` will be fired before `this._radios` storing. If I understand you right.

File: js/ui/collection/ui.collection_widget.edit.js [hunk 2]
```diff
@@ -481,18 +481,20 @@ var CollectionWidget = BaseCollectionWidget.inherit({
...
+    _processSelectableItem: function($itemElement, isSelected) {
+        var selectedItemClass = this._selectedItemClass();
 
-        if(this._isItemSelected(this._editStrategy.getNormalizedIndex($itemElement))) {
-            $itemElement.addClass(this._selectedItemClass());
-            this._setAriaSelected($itemElement, "true");
-        } else {
-            this._setAriaSelected($itemElement, "false");
-        }
+        isSelected ? $itemElement.addClass(selectedItemClass) : $itemElement.removeClass(selectedItemClass);
```
Comment (lukyanovas) [inline js/ui/collection/ui.collection_widget.edit.js]: Maybe better to use `$itemElement.toggleClass(this._selectedItemClass(), isSelected)`  ?

File: js/core/class.js [hunk 3]
```diff
@@ -44,6 +44,11 @@ var include = function() {
         name,
         i;
 
+    // NOTE: For ES6 classes. They don't have _includedCtors/_includedPostCtors
+    // properties and get them from the ancestor class.
+    classObj._includedCtors = classObj._includedCtors.slice(0);
+    classObj._includedPostCtors = classObj._includedPostCtors.slice(0);
```
Comment (churkin) [inline js/core/class.js]: @AlekseyMartynov How do you like that?

**Comment by churkin** [comment]
Ready to review

**Comment by AlekseyMartynov** [comment]
@churkin 
class.js changes LGTM

**Review by lukyanovas — APPROVED** [review]

**Review by jsdmitry — APPROVED** [review]

**Review by KsyushaM — APPROVED** [review]

**Pull request merged**